client/dashboard/task-list/index.js

```diff
@@ -59,108 +53,39 @@ class TaskDashboard extends Component {
-					'wooocommerce-admin'
-				),
-				before: hasProducts ? (
-					<i className="material-icons-outlined">check_circle</i>
-				) : (
-					<i className="material-icons-outlined">add_box</i>
-				),
-				after: <i className="material-icons-outlined">chevron_right</i>,
-				onClick: () => updateQueryString( { task: 'products' } ),
-				container: <Products />,
-				className: hasProducts ? 'is-complete' : null,
-				visible: true,
-			},
-			{
-				key: 'appearance',
-				title: __( 'Personalize your store', 'woocommerce-admin' ),
-				content: __( 'Create a custom homepage and upload your logo', 'wooocommerce-admin' ),
-				before: <i className="material-icons-outlined">palette</i>,
-				after: <i className="material-icons-outlined">chevron_right</i>,
-				onClick: () => updateQueryString( { task: 'appearance' } ),
-				container: <Appearance />,
-				className: customLogo && hasHomepage ? 'is-complete' : null,
-				visible: true,
-			},
-			{
-				key: 'shipping',
-				title: __( 'Set up shipping', 'woocommerce-admin' ),
-				content: __( 'Configure some basic shipping rates to get started', 'wooocommerce-admin' ),
-				before:
-					shippingZonesCount > 0 ? (
-						<i className="material-icons-outlined">check_circle</i>
-					) : (
-						<i className="material-icons-outlined">local_shipping</i>
-					),
-				after: <i className="material-icons-outlined">chevron_right</i>,
-				onClick: () => updateQueryString( { task: 'shipping' } ),
-				container: <Shipping />,
-				className: shippingZonesCount > 0 ? 'is-complete' : null,
-				visible: profileItems.product_types.includes( 'physical' ) || hasPhysicalProducts,
-			},
-			{
-				key: 'tax',
-				title: __( 'Set up tax', 'woocommerce-admin' ),
-				content: __(
-					'Choose how to configure tax rates - manually or automatically',
-					'wooocommerce-admin'
-				),
-				before: <i className="material-icons-outlined">account_balance</i>,
-				after: <i className="material-icons-outlined">chevron_right</i>,
-				onClick: () => updateQueryString( { task: 'tax' } ),
-				container: <Tax />,
-				visible: true,
-			},
-			{
-				key: 'payments',
-				title: __( 'Set up payments', 'woocommerce-admin' ),
-				content: __(
-					'Select which payment providers you’d like to use and configure them',
-					'wooocommerce-admin'
-				),
-				before: <i className="material-icons-outlined">payment</i>,
-				after: <i className="material-icons-outlined">chevron_right</i>,
-				onClick: () => updateQueryString( { task: 'payments' } ),
-				container: <Payments />,
-				className: paymentsCompleted ? 'is-complete' : null,
-				visible: true,
-			},
-		];
+	keepTaskCard() {
+		recordEvent( 'tasklist_completed', {
+			action: 'keep_card',
+		} );
+
+		this.setState( { promptShown: true } );
+	}
+
+	hideTaskCard( action ) {
+		recordEvent( 'tasklist_completed', {
+			action,
+		} );
+		this.props.updateOptions( {
+			[ 'woocommerce_task_list_hidden' ]: 'yes',
```

Comment:
Minor, but I don't think the brackets are needed here.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/3021/commits/50b40260cb5c93bdfcb532d6ae891f26ab663d32.